### PR TITLE
Add keystone options to client

### PR DESCRIPTION
**Pull request by rbrndt**
If keystone options are not configured, will
maintain current behavior

### CODE REVIEW

File: monascaclient/keystone.go
Comment (mhoppal) [inline monascaclient/keystone.go]: awesome work on this!!! can we have the option to read from env as well?

Comment (mhoppal) [reply monascaclient/keystone.go]: https://github.com/monasca/smoke-test/blob/master/smoke-test.go#L205

File: monascaclient/client.go
Comment (mhoppal) [inline monascaclient/client.go]: pass by value?

**Review by mhoppal — APPROVED** [review]
👍  testing once wifi decides to work and will merge if it works

File: monascaclient/keystone.go
Comment (craigbr) [inline monascaclient/keystone.go]: Personally, I would rather this just returned the token and the code in client.go did things like headers.Add(). Seems strange to call this and have it overwrite the headers in the client

File: monascaclient/keystone.go
Comment (craigbr) [inline monascaclient/keystone.go]: Remove

File: monascaclient/keystone.go
Comment (rbrndt) [reply monascaclient/keystone.go]: Where in the client.go did you think this should go? I don't see an obvious place.

**Pull request merged**
